index.html

```diff
@@ -5,9 +5,59 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="main.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;500;600;700;800&family=Mulish:wght@300;400;500;600;700&display=swap" rel="stylesheet">
     <title>Screeno</title>
 </head>
 <body>
-
+    <section class="main-section">
+        <div class="main-content">
+            <nav class="main-nav">
+                <img class="main-logo" src="img/title.png" alt="logo">
+                <ul class="main-nav-ul">
+                    <li class="main-nav-li">
```

Comment:
jeśli to jest nawigacja to są w niej linki, zatem w środku jeszcze pewnie przydałoby się jakieś <a> ale na potrzeby testowe nie musi być chociaż mógłbyś zrobić nawet <a href="#"> żeby zaznaczyć że to link, i wtedy nie musisz w klasie pisać ze to li, bardziej jaką funkcje to spełnia czyli main-nav-link skoro to link